docs/coding-guidelines/javascript.md

```diff
@@ -559,8 +562,8 @@ prop = object[ 'default' ];
 prop = object[ 'key-with-hyphens' ];
 ```
 
-That said, avoid accessing nested properties through a chain of dot notation as this can lead cause access errors.
-Instead, use Lodash's [`get`](https://lodash.com/docs#get) function. It will safely handle cases where a property or object is missing at any point in the nesting chain.
+For nested properties, avoid using Lodash's [`get`](https://lodash.com/docs#get) function.
+Instead, you can use the standard optional chaining syntax (`?.`) after properties that may be missing. That will safely handle cases where a property or object is missing at any point in the nesting chain.
```

Comment:
Should we also add a sentence to suggest nullish coalescing usage here?

Reply:
Good idea! Done.